public/pages/Overview/components/Widgets/DetectorsWidget.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiBasicTableColumn, EuiButton } from '@elastic/eui';
+import React from 'react';
+import { DetectorItem } from '../../models/interfaces';
+import { TableWidget } from './TableWidget';
+import { WidgetContainer } from './WidgetContainer';
+
+const columns: EuiBasicTableColumn<DetectorItem>[] = [
+  {
+    field: 'detectorName',
+    name: 'Detector name',
+    sortable: true,
+    align: 'left',
+  },
+  {
+    field: 'status',
+    name: 'Status',
+    sortable: false,
+    align: 'left',
+  },
+  {
+    field: 'logTypes',
+    name: 'Log types',
+    sortable: true,
+    align: 'left',
+    render: (logTypes: string[]) => <p>{logTypes[0]}</p>,
+  },
+];
+
+export interface DetectorsWidgetProps {
+  items: DetectorItem[];
+}
+
+export const DetectorsWidget: React.FC<DetectorsWidgetProps> = ({ items }) => {
+  const actions = React.useMemo(
+    () => [
+      <EuiButton href="">View all detectors</EuiButton>,
+      <EuiButton href="">Create detector</EuiButton>,
+    ],
+    []
+  );
+
```

Comment:
This may be naive of me, but why memoize this?

Reply:
to avoid re-creating the button component for every re-render